samila/genimage.py

```diff
@@ -145,6 +145,12 @@ def generate(
                 elif self.generate_mode == GenerateMode.X2_VS_F2.value:
                     data1_ = point[1]
                     data2_ = self.function2(point[0], point[1]).real
+                elif self.generate_mode == GenerateMode.F1F2_VS_F1.value:
+                    data1_ = self.function1(point[0], point[1]).real * self.function2(point[0], point[1]).real
+                    data2_ = self.function1(point[0], point[1]).real
+                elif self.generate_mode == GenerateMode.F1F2_VS_F2.value:
+                    data1_ = self.function1(point[0], point[1]).real * self.function2(point[0], point[1]).real
+                    data2_ = self.function2(point[0], point[1]).real
```

Comment:
Not sure if these are food atomic modes. If we add this we should add `F1_VS_F1F2`, `F1_VS_F2F1`, and `F1+F2_VS_F1` etc. Therefore I think that's not a good idea to add this mode.

Reply:
If it doesn't produce a beautiful mode, it is also not harmful.

Reply:
I don't know. I wanted this set to be atomic, or at least closed under the operations we're defining (including all cases).

Reply:
I understand your point, and if you're okay with it, I will add `F1_VS_F1F2` and `F1_VS_F2F1` in the next version of Samila. I generated this image using the `F1F2_VS_F1` mode:


![test](https://github.com/user-attachments/assets/025480f3-52ec-4e2a-a827-e2e94a0f1d93)

Reply:
OK, that's beautiful. I guess that's fine then.